app/presenters/publishing_api/world_location_news_article_presenter.rb

```diff
@@ -0,0 +1,70 @@
+module PublishingApi
+  class WorldLocationNewsArticlePresenter
+    include UpdateTypeHelper
+    attr_reader :item
+    attr_reader :update_type
+
+    def initialize(item, update_type: nil)
+      @item = item
+      @update_type = update_type || default_update_type(item)
+    end
+
+    def content_id
+      item.content_id
+    end
+
+    def content
+      content = BaseItemPresenter.new(item).base_attributes
+      content.merge!(
+        description: item.summary,
+        details: details,
+        document_type: "news_article",
```

Comment:
I think we should make this `world_location_news_article` as it we have a `news_article` `schema_name` on the way with the next format. All being well as we migrate news articles we can combine the two formats having established that they are identical apart from the presence of the worldwide organisation link.